modules/reference/pages/rpk/rpk-cluster/rpk-cluster-self-test-start.adoc

```diff
@@ -3,6 +3,8 @@
 
 Starts one or more benchmark tests on one or more nodes of the cluster.
 
+NOTE: Redpanda self-test runs benchmarks of disk and network hardware that consume significant system resources. Do not start self-test if large workloads are already running on the system. 
```

Comment:
Correct me if my understanding is wrong -- isn't the self test itself the one that consumes significant resources? This makes it sound like it's the disk/network hardware

Reply:
Rephrased to make it more simple, and removed the specific tests to make it a bit more future proof.